aimmo-game/tests/simulation/test_game_state.py

```diff
@@ -50,6 +51,7 @@ def test_add_avatar(self):
         self.assertIn(7, self.av_manager._avatars_by_id)
         self.assertEqual(self.av_manager._avatars_by_id[7].location, Location(10, 10))
 
+    @unittest.skip('Doesn\'t test fog of war.')
```

Comment:
Think this should be `test_fog_of_war_applied` (ie: check that `GameState` calls `apply_fog_of_war` and then passes the result to `decide_action`).